### PR TITLE
First-Upload

### PR DESCRIPTION
**Pull request by SaadMahi**

### CODE REVIEW

**Comment by SaadMahi** [comment]
resize your logo!

File: style.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  margin-bottom: 0;
+  padding-bottom: 1em;
+}
+
+.logo:hover {
+  cursor: pointer;
+}
+
+.content {
+  /*background-image: url(https://gallery.yopriceville.com/var/resizes/Backgrounds/Cinema_Background.jpg?m=1399672800);*/
```
Comment (ManSOorcode) [inline style.css]: remove this comment .

File: style.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+nav {
+  position: absolute;
+  text-align: left;
+  top: 100%;
+  left: 0;
+  background: #242424;
+  width: 100%;
+  transform-origin: top;
+  transform: scale(1, 0);
+  transition: transform 400ms ease-in-out;
```
Comment (ManSOorcode) [inline style.css]: group similar property so other developer can read your document easily.

Comment (SaadMahi) [reply style.css]: please be more specific, I'm unable to understand the issue.

**Review by SaadMahi — COMMENTED** [review]
comment removed from style.css as ManSOorcode specified.

File: style.css (hunk 1 above)
Comment (SaadMahi) [reply style.css]: removed

**Comment by OSAMA478** [comment]
group similar properties according to there type just like mentioned below @SaadMahi 
.module {
  background: rgba(198, 148, 72, 0.75);
  color: #222;
  opacity: 1;

  border: 2px solid black;
  border-radius: 4px;

  font-family: sans-serif;
  font-size: 0.9rem;
  line-height: 1.4;
  
  position: relative;
  display: flex;
  flex-direction: column;
  clear: both;
  overflow: hidden;

  margin: 0 0 2rem 0;
  padding: 1rem;
  
  transform: opacity 0.2s ease-in-out;
  transform-origin: right center;
}

File: style.css [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+  nav a::before {
+    content: "";
+    display: block;
+    height: 5px;
+    background: crimson;
+    position: absolute;
+    bottom: -0.75em;
+    left: 0;
+    right: 0;
```
Comment (OSAMA478) [inline style.css:251]: use short hand property (inset) here and also different places too in this code...

**Review by OSAMA478 — APPROVED** [review]
apart from grouping properties and using shorthand properties, everything looks good to me

File: style.css (hunk 2 above)
Comment (ManSOorcode) [reply style.css]: as *@OSAMA478* provide you example do this way  it's called grouping similar property. 
***If you didn't understand just refer this article :***
https://css-tricks.com/different-logical-ways-group-css-properties

**Review by ManSOorcode — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (SaadMahi) [reply style.css:251]: okay great will apply these changes, thanks osama and mansoor for your valuable reviews.

File: style.css (hunk 2 above)
Comment (SaadMahi) [reply style.css]: great informative link, will apply those changes. thanks mansoor bhai.

**Review by SaadMahi — COMMENTED** [review]
grouped-by-type css style

**Review by SaadMahi — COMMENTED** [review]
all above changes made, except short hand property as while using short hand property my anchor link animation doesn't responds the way i want them to.

**Review by murtazshaikh — APPROVED** [review]
LGTM!

**Review by syedabdullahali — APPROVED** [review]

**Comment by SaadMahi** [comment]
All changes are resolved and ready to push to main branch

**Comment by SaadMahi** [comment]
pushing the accepted changes

**Review by SaadMahi — COMMENTED** [review]
all changes are reviewed and acceptable

**Pull request merged**